xrdsst/controllers/base.py

```diff
@@ -87,15 +87,18 @@ def get_api_key(self, conf, security_server):
 
     @staticmethod
     def init_logging(configuration):
+        log_file_name = configuration["logging"][0]["file"]
+        log_level = configuration["logging"][0]["level"]
         try:
-            log_file_name = configuration["logging"][0]["file"]
-            if not os.path.isfile(log_file_name):
-                raise FileNotFoundError
-            logging.basicConfig(filename=log_file_name,
-                                level=configuration["logging"][0]["level"],
-                                format='%(name)s - %(levelname)s - %(message)s')
-        except FileNotFoundError as err:
-            print("Configuration file \"" + log_file_name + "\" not found: %s\n" % err)
+            exists = os.path.exists(log_file_name)
+            if exists and os.path.isdir(log_file_name):
```

Comment:
If non-existing path is given, nothing is reported to the user and nothing gets logged